### PR TITLE
Fix custom api endpoint

### PR DESCRIPTION
**Pull request by dmyerscough**
Tested by @bocig

### CODE REVIEW

File: salt/returners/hipchat_return.py [hunk 1]
```diff
@@ -125,6 +125,7 @@ def _get_options(ret=None):
     profile_attrs = {'from_jid': 'from_jid',
                      'api_key': 'api_key',
                      'api_version': 'api_key'
```
Comment (cachedout) [inline salt/returners/hipchat_return.py]: Missing a comma.

**Comment by cachedout** [comment]
@dmyerscough There's a small syntax error. Could you please fix it and we'll get this merged? Thanks!

**Comment by bocig** [comment]
Hey @dmyerscough & @cachedout ,

I've just checked the latest update. It seems the url does Not get picked up when it is specified in:
```
    profile_attrs = {'from_jid': 'from_jid',
                     'api_key': 'api_key',
                     'api_version': 'api_key',
                     'api_url': 'api_url',
                     }```

It does work when passed to default attrs. Briefly looking at the code it seems there are places where the default url is still hard coded, needs a bit more cleanup. Thanks!

**Comment by dmyerscough** [comment]
@cachedout can we revert my old change, I don't have a test system so @bocig can you tell me what change you made earlier?

**Comment by bocig** [comment]
@cachedout , I've submitted a pull request (#29158) with my updates. Tested and working with custom hipchat api endpoint.

**Pull request merged**